src/gui/MainWindow.cpp

```diff
@@ -822,18 +826,24 @@ void MainWindow::onActiveDocumentChanged(const int newIndex){
             for(QAction * action:singleViewAct) action->setChecked(false);
             singleViewAct[documents[activeDocumentId]->getDisplayGrid()->getActiveDisplayId()]->setChecked(true);
         }
-
+    }
+    else if (displayGrid == nullptr && documentArea->count() > 1) {
+        objectTreeWidgetDockable->clear();
+        objectPropertiesDockable->clear();
+        statusBarPathLabel->setText("");
+        activeDocumentId = -1;
     }
 }
 
 void MainWindow::tabCloseRequested(const int i)
 {
     documentArea->removeTab(i);
-    if (documentArea->currentIndex() == -1){
+    if (documentArea->currentIndex() == -1 || QString::compare(documentArea->tabText(documentArea->currentIndex()), "Quick Start", Qt::CaseSensitive) == 0) {
```

Comment:
Don't test for the name of the tab. This isn't a criterion. There may be a model with file name "Quick Start".

Reply:
https://doc.qt.io/qt-5/qfilesystemmodel.html#fileName this is the model you are referring?

Reply:
btw in which Model, the file names are actually stored? I can't find one.

Reply:
> There is no model. The file path is stored in the Document object. The file name part is used as tab name too.

But `Quick Start` tab name is not stored as file name in `filePath` here.

Reply:
I have an idea to solve this issue. Instead of freeing the memory of each `documents` after closing the application, should we free each `documents` whenever we close a tab? But this will leads to a change in value for `documentsCount` too because we have to decrement it every time when we close the tab and free the memory. But I saw in comment `// Total number of documents ever opened in application's life time. This is not decreased when closing documents.`

Reply:
Maybe, but this would be something for another PR.